engine/script/src/test/test_script.cpp

```diff
@@ -215,9 +215,9 @@ TEST_F(ScriptTest, TestRandom)
 {
     int top = lua_gettop(L);
     ASSERT_TRUE(RunString(L, "math.randomseed(123)"));
-    ASSERT_TRUE(RunString(L, "assert(math.random(0,100) == 1)"));
     ASSERT_TRUE(RunString(L, "assert(math.random(0,100) == 58)"));
-    ASSERT_TRUE(RunString(L, "assert(math.abs(math.random() - 0.70419311523438) < 0.0000001)"));
```

Comment:
We don't want a test for avoiding the previous behavior?

(The current test only tests the actual values, not the series of values described in the issue #3753)